### PR TITLE
Fix playback ui not updating during player seek backfill

### PR DESCRIPTION
**Pull request by snosenzo**
**User-Facing Changes**

 - Fix playback UI not updating while loading

**Description**

Looks like the `isPlaying` variable change when play/pause is called is never being emitted because the player is in `seek-backfill` state. I've updated play/pause to `queueEmitState` when they don't call `setState`. This will update the UI while the Player is in `seek-backfill`. 

FG-4474

### CODE REVIEW

**Review by foxymiles — APPROVED** [review]

**Review by jtbandes — APPROVED** [review]
What could possibly go wrong?

**Comment by jtbandes** [comment]
Can this be added to IterablePlayer unit tests?

**Comment by snosenzo** [comment]
> Can this be added to IterablePlayer unit tests?

@jtbandes Added one for setting play, but struggling to get the pause one working. For the final `store.reset` if I put `reset(2)` it gives me an error that it's getting 3 and if i put `reset(3)` it's getting a timeout. Any thoughts/suggestions?

File: packages/studio-base/src/players/IterablePlayer/IterablePlayer.ts [hunk 1]
```diff
@@ -1047,6 +1063,7 @@ export class IterablePlayer implements Player {
     await this.#playbackIterator?.return?.();
     this.#playbackIterator = undefined;
     await this.#iterableSource.terminate?.();
+    this.#resolveIsClosed(true);
```
Comment (jtbandes) [inline packages/studio-base/src/players/IterablePlayer/IterablePlayer.ts:1066]: maybe leave out the `true` i.e. use `Promise<void>`?

**Review by jtbandes — APPROVED** [review]

**Pull request merged**